### PR TITLE
Add vdom recipe

### PR DESCRIPTION
**Pull request by gnestor**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/vdom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - vdom
+
+about:
+  home: "https://github.com/nteract/vdom"
+  license: "BSD"
+  license_family: "BSD"
+  license_file: ""
```
Comment (chrisburr) [inline recipes/vdom/meta.yaml]: This field needs to be filed:
```suggestion
  license_file: LICENSE
```

If it's not present in the source tarball you can download it from here and put it in the recipe directory
https://github.com/nteract/vdom/blob/master/LICENSE

File: recipes/vdom/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  summary: "VDOM for Python"
+  doc_url: ""
```
Comment (chrisburr) [inline recipes/vdom/meta.yaml]: These should be filled or removed

File: recipes/vdom/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - jsonschema
+    - python
+
+test:
+  imports:
+    - vdom
+
+about:
+  home: "https://github.com/nteract/vdom"
+  license: "BSD"
```
Comment (chrisburr) [inline recipes/vdom/meta.yaml]: This should specify which varient of the BSD licence applies:
```suggestion
  license: BSD-3-Clause
```

File: recipes/vdom/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: "d43fb8e92e64e1cabbdb93b4f213b95145db0e843371239b3a8bf91dbdf65159"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - ipython
+    - jsonschema
```
Comment (chrisburr) [inline recipes/vdom/meta.yaml]: Do you need `jsonschema` and `ipython` in `host`? Normally `python` and `pip` is enough

File: recipes/vdom/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d43fb8e92e64e1cabbdb93b4f213b95145db0e843371239b3a8bf91dbdf65159"
+
+build:
+  number: 0
```
Comment (chrisburr) [inline recipes/vdom/meta.yaml]: This looks like it could be noarch?
```suggestion
  number: 0
  noarch: python
```

Comment (gnestor) [reply recipes/vdom/meta.yaml]: I'm not sure, it's just a vanilla Python library so I generated the recipe using conda skeleton.

File: recipes/vdom/meta.yaml (hunk 4 above)
Comment (gnestor) [reply recipes/vdom/meta.yaml]: Good catch!

**Comment by gnestor** [comment]
I just noticed that I created this PR from the wrong origin. I'm going to close this PR and open a new one...

**Pull request closed without merging**